docs/pages/compatibility.md

```diff
@@ -16,18 +16,20 @@ tags:
   <tr><td>Firefox</td></tr>
   <tr><td>Safari</td></tr>
   <tr><td>Opera</td></tr>
-  <tr><td>Mobile Safari</td></tr>
+  <tr><td>Mobile Safari<sup>1</sup></td></tr>
   <tr><td>IE Mobile</td></tr>
   <tr>
     <td>Internet Explorer</td>
     <td class="works">Versions 9+</td>
   </tr>
   <tr>
     <td>Android Browser</td>
-    <td class="works">Versions 2.3+</td>
+    <td class="works">Versions 4.4+</td>
   </tr>
 </table>
 
+<sup>1</sup>iOS 7+ is actively supported but with some known bugs.
```

Comment:
Does _all_ iOS 7+ versions have bugs ? ;)
Please add the concerned versions and links to the issues.

Reply:
> Please add the concerned versions and links to the issues.

Well, this would be many. iOS Safari <= 10 has flexbugs issues and there are also numerous issues with Safari 9. And both are officially not supported by Apple anymore. That's why I wanted to make a cut. We can not support all these old and problematic Safari releases (which also did not support required form fields and so on).

So "some known bugs" is not right in my opinion.

https://www.reddit.com/r/web_design/comments/3la04p/psa_safari_on_ios9_has_a_media_query_bug/

Safari is the new IE for us in the frontend ;-)

Reply:
> Any time a web browser is tied to the release cycle of an OS it's going to be bad. IE suffered from this problem in the past and Safari both on iOS and the main OS suffers from not being on an independent release cycle. It's really stupid.

Reply:
What do you think @colin-marshall @SassNinja @JeremyEnglert ?
Foundation is known for its compatibility with older browsers (even if it's a bit less true now with flex & XY grid).

See the discussion above https://github.com/zurb/foundation-sites/pull/11021#issuecomment-373955647